wrapper.html

```diff
@@ -4,7 +4,7 @@
 	{% endif %}>
 
 <head>
-
+	{% if page.custom_fields.enable_google_optimize == 'Y' %}

```

Comment:
Is the actual Optimize script loaded via Google Tag Manager? If so good to go here.

Reply:
I'm assuming so! This is just covering the anti flicker snippet

Reply:
@sukhada shouldn't this actually be lowercase check?